docker-compose.yml

```diff
@@ -0,0 +1,273 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# Basic Airflow cluster configuration for CeleryExecutor with Redis and PostgreSQL.
+#
+# WARNING: This configuration is for local development. Do not use it in a production deployment.
+#
+# This configuration supports basic configuration using environment variables or an .env file
+# The following variables are supported:
+#
+# AIRFLOW_IMAGE_NAME           - Docker image name used to run Airflow.
+#                                Default: apache/airflow:2.2.5
```

Comment:
how about using the same version ([2.1.3](https://airflow.apache.org/docs/apache-airflow/2.1.3/start/docker.html)) as what we are using in production. 

I also specify that in the[ problem description](https://github.com/EQWorks/ws-problems/pull/238/files#diff-9afed27252786b64eb016177aa404d03cebd31c28ea46459a85b7e65fb844669R48).

But should be good if you think using this one (currently the latest one) is better.

Reply:
agree, should use the same version 2.1.3

we _could_ leave it open for the candidate (described in problems) if they wish to leverage any particular feature from the newer versions (such as 2.2.5), they'd need to make necessary `docker-compose.yml` changes so that it'd run properly

Reply:
it may be better to specify a fixed version. Two reasons:

1. some module/packages in Airflow may not be backward compatible unless we use the same version when we are reviewing their work, which will add a significant workload to us. 
2. the code I wrote may not work in a newer version.

Reply:
sounds good

fixed with 2.1.3 in 88e7bad